### DISCUSSION TITLE
Does liboqs support hybrid algorithms?

### DISCUSSION THREAD

**Discussion by T-449** [open-quantum-safe/liboqs#1910] [Q&A] [answered]
I searched for the hybrid algorithms in the src but all I find are the basic algorithms and not the hybrid ones. The supported algorithms section also doesn't show any hybrid algorithm listed. But I remember support for those algorithms on the OQS-OpenSSH doc and since it uses liboqs, i don't know why the code isn't there in the liboqs repo.

**Comment by baentsch** [comment, marked as answer]
No. Hybrid algs are only implemented in downstream projects such as oqsprovider for openssl and oqs-openssh.

**Reply by T-449** [reply]
Thanks!!
